TFT/src/User/API/Vfs/vfs.c

```diff
@@ -137,9 +137,7 @@ char * isSupportedFile(char * filename)
   char * extPos = strrchr(filename, '.');  // check last "." in the name where extension is supposed to start
 
   if (extPos != NULL && extPos[1] != 'g' && extPos[1] != 'G')
-  {
     extPos = NULL;
-  }
```

Comment:
Please make up your mind, should brackets be used or not at single line "if"...

Reply:
I never used brackets for one line "if" or "if else". Furthermore, all the file is aligned with that convention